src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/reduce.cpp

```diff
@@ -19,6 +19,10 @@ std::vector<std::vector<ov::test::InputShape>> inputShapes_dyn = {
     {{{{1, 5}, 19, {1, 5}, {1, 10}}, {{2, 19, 2, 2}, {2, 19, 2, 9}}}},
 };
 
+std::vector<std::vector<ov::test::InputShape>> inputShapes_3D_fuse_dyn = {
+    {{{{1, 5}, 19, {1, 10}}, {{1, 19, 2}, {1, 19, 9}}}},
+};
+
```

Comment:
Could we add 2D test case to cover corner case?

Reply:
Thanks Chenhu for the comment! After the reducing the 3D case is actually the 2D fusing case, that can reproduce the issue.